src/renderers/dom/shared/SVGDOMPropertyConfig.js

```diff
@@ -264,7 +264,9 @@ var ATTRS = {
   xlinkType: 'xlink:type',
   xmlBase: 'xml:base',
   xmlLang: 'xml:lang',
+  xmlNS: 'xmlns',
```

Comment:
You are right @iamdustan, capitalisation happens after `:` or `-` and this attribute (`xmlns`) seems to be al little weird (but this is a _spec_ problem). I just followed the point 3 from [this comment](https://github.com/facebook/react/pull/6471#issuecomment-208447923) and seems coherent to me.